src/components/pages/CommunityPage/CommunityPage.tsx

```diff
@@ -53,6 +63,27 @@ export default function CommunityPage(): JSX.Element {
     teamCtrsSets[CountTheSets].push(element);
   });
 
+  function MemberCard({ member, index }) {
+    const githubUsername = member.githubUsername?.[0] || member.apacheId;
```

Comment:
@visortelle btw. this might make a wrong choice for some users. It might be better to not assume that github username matches the apache id. 
Every committer should go to [Whimsy](https://whimsy.apache.org/roster/committer/__self__) and add their GitHub id there.

Reply:
I guess the default might be fine, but we should send emails to the ones that haven't added their GitHub ID in Whimsy if they want to have their GitHub link shown on the website.

Reply:
Actually, the default isn't great, I'd rather have everyone opt-in by adding their GitHub id explicitly.

Reply:
Agree. In this case, we should render a regular div instead of a link that refers to the GitHub profile.

Reply:
fixed